### PR TITLE
Add GUIX Installation Steps

### PR DESCRIPTION
**Pull request by asddsaz**
**Description**: Aye, I haven' used `middleman` in a while so there may be some issues but this should add GNU Guix's installation steps to the website. As always, edits from maintainers are welcome.
**Has this been discussed before**: All of the information for this PR is listed here: #333

### CODE REVIEW

File: data/distro.yml
Comment (AsciiWolf) [inline data/distro.yml]: Does it also install the GNOME Software (or KDE Discover) Flatpak plugin?

Comment (allanday) [reply data/distro.yml]: I'm waiting for this question to be answered before merging.

Comment (asddsaz) [reply data/distro.yml]: I do not believe so.
You can see all packages here: https://www.gnu.org/software/guix/packages/

I do not see `gnome-software-plugin-flatpak` under [/software/guix/packages/G/](https://www.gnu.org/software/guix/packages/G/)

Comment (AsciiWolf) [reply data/distro.yml]: Hmm, I don't see the `gnome-software` package at all under /software/guix/packages/G/.

Comment (AsciiWolf) [reply data/distro.yml]: What is the software center GUI used in GNU Guix and does it include Flatpak support?

Comment (AsciiWolf) [reply data/distro.yml]: Hmm, it seems that [this](http://www.ocsmag.com/wp-content/uploads/2016/07/guix-in-emacs.png) is the package manager "GUI" that GNU Guix actually use. Does it have any Flatpak support?

Comment (AsciiWolf) [inline data/distro.yml]: I have just noticed `<span class="unselectable">$</span> #`. This is wrong, it should be `<span class="unselectable">#</span>`.

I will be happy to merge this PR after this is fixed and [my previous question](https://github.com/flatpak/flatpak.github.io/pull/337#discussion_r321697811) is answered (and information that Flatpak installation via GUI is not possible is added, if it really is not possible) + merge conflicts are resolved. ;-)

Comment (AsciiWolf) [reply data/distro.yml:489]: @asddsaz @lionirdeadman What is the current state of Flatpak support in GUIX?

Comment (AsciiWolf) [reply data/distro.yml]: @asddsaz @lionirdeadman What is the current state of Flatpak support in GUIX?

File: data/distro.yml
Comment (AsciiWolf) [inline data/distro.yml]:
```suggestion
        <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
        <p>Note: graphical installation of Flatpak apps may not be possible with GNU Guix.</p>
```

File: data/distro.yml
Comment (AsciiWolf) [inline data/distro.yml]:
```suggestion
          <span class="unselectable">#</span> guix install flatpak
```

**Review by AsciiWolf — CHANGES_REQUESTED** [review]

File: data/distro.yml
Comment (AsciiWolf) [reply data/distro.yml]: See the suggested changes.

Comment (AsciiWolf) [reply data/distro.yml]: See the suggested changes.

File: data/distro.yml
Comment (AsciiWolf) [inline data/distro.yml]:
```suggestion
    </ol>

```

File: data/distro.yml
Comment (AsciiWolf) [inline data/distro.yml:791]: What about Flathub? Is it added out-of-box in Guix or why is this step missing?

**Review by AsciiWolf — CHANGES_REQUESTED** [review]

File: data/distro.yml
Comment (AsciiWolf) [inline data/distro.yml]:
```suggestion
- name: "GNU Guix"
```

File: data/distro.yml
Comment (AsciiWolf) [reply data/distro.yml:791]: Any chance someone could answer the question from previous comment? :-)

Comment (AsciiWolf) [reply data/distro.yml:791]: I have tested Guix (by the way, it's Guix, not GUIX) and it looks like that it has no support for graphical management of software at all.

**Review by AsciiWolf — APPROVED** [review]

**Pull request merged**
